cli/src/api/mod.rs

```diff
@@ -171,9 +171,9 @@ impl PhylumApi {
             HeaderValue::from_str(&format!("Bearer {}", tokens.access_token)).unwrap(),
         );
         headers.insert("Accept", HeaderValue::from_str("application/json").unwrap());
-        headers.insert("version", HeaderValue::from_str(version).unwrap());
```

Comment:
I am fairly confident that this header is not being used or recorded anywhere and I think the `User-Agent` is a much more natural place to include the CLI version.

Cc @DanielJoyce in case I am incorrect on that assumption.

Reply:
The version was added a while back to help debug when a customer was using a _super old_ version. I don't think it's as much of a worry now, but it did end up being useful at one point. If it's in the UA string, I think that's good too.